tests/test_csc.py

```diff
@@ -23,15 +23,15 @@
 import unittest
 
 from lsst.ts import salobj
-from lsst.ts.dimm import dimm_csc
+from lsst.ts.dimm.dimm_csc import DIMMCSC, __version__
```

Comment:
I strongly suggest deleting the `dimm_csc`. That avoids using internal details of the package (the name of the module that defines the public class DIMMCSC, which you might want to change someday).

Also I feel it is confusing to import `__version__` as a lone symbol, since it is not at all obvious where it comes from. The simplest fix for that is to change this import to:
```
from lsst.ts import dimm
```
and then use `dimm.DIMMCSC` and `dimm.__version__` in the code below.